4001-5000/LIT4872MaqdemaWangel.xml

```diff
@@ -52,31 +52,92 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         <revisionDesc>
             <change who="DR" when="2018-01-11">Created entity</change>
             <change who="SD" when="2020-01-07">Added refrerence, title translation and abstract</change>
+            <change who="ABr" when="2020-11-13">Added Introduction with incipit and explicit</change>
+            <change who="ABr" when="2020-11-14">Added titles, incipit and explicit to Chapters 1-7</change>
         </revisionDesc>
     </teiHeader>
     <text xml:base="https://betamasaheft.eu/">
         <body>
-            <div type="bibliography">
-                <listBibl type="secondary">
-                    <bibl>
-                  <ptr target="bm:Zuurmond1989EvangeliumMarc"/>
-                  <citedRange unit="page">14-16</citedRange>
-               </bibl>
-               <bibl>
-             <ptr target="bm:Cowley1977NT"/>
-             <citedRange unit="page">144-165</citedRange>
-          </bibl>
-                </listBibl>
+          <div type="bibliography">
+          <listBibl type="secondary">
+            <bibl>
+          <ptr target="bm:Zuurmond1989EvangeliumMarc"/>
+            <citedRange unit="page">14-16</citedRange>
+            </bibl>
+            <bibl>
+              <ptr target="bm:Cowley1977NT"/>
+              <citedRange unit="page">144-165</citedRange>
+            </bibl>
+            </listBibl>
             </div>
-            <div type="edition">
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter1"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter2"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter3"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter4"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter5"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter6"/>
-                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Chapter7"/>
+        <div type="edition">
+            <note>The following division in textual units is based on
+              <bibl><ptr target="bm:Cowley1977NT"/><citedRange unit="page">147-160</citedRange></bibl>and on
```

Comment:
please leave a space before "and": `</bibl> and`

Reply:
ok